torch/csrc/lazy/ts_backend/ts_eager_fallback.cpp

```diff
@@ -107,8 +108,8 @@ static std::unordered_map<std::string, ::torch::lazy::Counter*>
     _eager_fallback_counters;
 
 bool force_eager_fallback(c10::Symbol op) {
-  static char* force_str = std::getenv("LTC_FORCE_FALLBACK");
-  if (force_str != nullptr) {
+  auto force_str = getLTCForceFallback();
+  if (!force_str.empty()) {
```

Comment:
@yxd886 this is the code that actually implements the force fallback.

you could modify it to accept a list of ops separated by a comma or something, but currently it only supports a single op, and its both more complex and less efficient to set it up for a long list of many ops.

if you want another mode that just falls back all the time, you could add an env for that and make this function just always return true based on that env.